src/http/representation/RepresentationMetadata.ts

```diff
@@ -330,4 +379,16 @@ export class RepresentationMetadata {
       this.set(CONTENT_LENGTH_TERM, toLiteral(input, XSD.terms.integer));
     }
   }
+
+  /**
+   * Shorthand for the ContentType as an object (with parameters)
+   */
+  public get contentTypeObject(): ContentType | undefined {
```

Comment:
Would move this to be below the contentType getter/setter as they are related. Would perhaps also add a setter just for completeness.